<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>Thomas Klammsteiner</title> <meta name="author" content="Thomas Klammsteiner"> <meta name="description" content="microbes | environment | data "> <meta name="keywords" content="microbiome, data science, microbial communities, insect biotechnology, insects, waste management, anaerobic digestion, bioinformatics"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?19f3075a2d19613090fe9e16b564e1fe" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/icon.jpg?4ad390f64e9fd1b08f8da0b3b05a4fcb"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://tklammsteiner.github.io/"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?e74e74bf055e5729d44a7d031a5ca6a5" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?96d6b3e1c3604aca8b6134c7afdd5db6"></script> <script src="/assets/js/dark_mode.js?9b17307bb950ffa2e34be0227f53558f"></script> </head> <body class="fixed-top-nav sticky-bottom-footer"> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link" href="/">about<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog</a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications</a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects</a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <div class="profile float-right"></div> <h1 class="post-title"> <span class="font-weight-bold">Thomas</span> Klammsteiner </h1> <p class="desc"><b>microbes | environment | data</b></p> <div class="address"> <p>Postdoctoral researcher<br> Department of Ecology<br> University of Innsbruck, Innsbruck, Austria</p> </div> <div class="social"> <div class="contact-icons"> <a href="mailto:%74%68%6F%6D%61%73.%6B%6C%61%6D%6D%73%74%65%69%6E%65%72@%75%69%62%6B.%61%63.%61%74" title="email"><i class="fas fa-envelope"></i></a> <a href="https://scholar.google.com/citations?user=jI_Q6tgAAAAJ&amp;hl=de" title="Google Scholar" rel="external nofollow noopener" target="_blank"><i class="ai ai-google-scholar"></i></a> <a href="https://www.researchgate.net/profile/Thomas-Klammsteiner/" title="ResearchGate" rel="external nofollow noopener" target="_blank"><i class="ai ai-researchgate"></i></a> <a href="https://github.com/tklammsteiner" title="GitHub" rel="external nofollow noopener" target="_blank"><i class="fab fa-github"></i></a> <a href="https://www.linkedin.com/in/tklammsteiner" title="LinkedIn" rel="external nofollow noopener" target="_blank"><i class="fab fa-linkedin"></i></a> </div> <div class="contact-note"> Feel free to reach out via mail. </div> </div> </header> <article> <div class="clearfix"> <br> <p>I am a microbiologist specializing in microbial ecology, with a strong focus on microbiome research related to digestion and microbe-host interactions. I am broadly interested in biological data science and apply advanced analytical approaches to better understand complex biological systems. Through my research, I aim to contribute new insights and practical solutions that bridge fundamental science and real-world applications.</p> </div> <br> <h3><a href="/news/" style="color: inherit;">news</a></h3> <div class="news"> <div class="table-responsive" style="max-height: 60vw"> <table class="table table-sm table-borderless"> <tr> <th scope="row" style="width: 20%">May 30, 2025</th> <td> <a class="news-title" href="/news/2025-05-30-publication-iscience/">New publication: Klammsteiner et al. (2025) iScience 28(7), 112794</a> </td> </tr> <tr> <th scope="row" style="width: 20%">May 21, 2025</th> <td> <a class="news-title" href="/news/2025-05-21-publication-j-insects-food-feed-bugbook-data/">New publication: Smetana et al. (2025) J Insects Food Feed, in press</a> </td> </tr> <tr> <th scope="row" style="width: 20%">May 16, 2025</th> <td> <a class="news-title" href="/news/2025-05-16-publication-j-environ-manag/">New publication: Gómez-Brandón et al. (2025) J Environ Manag 386, 125774</a> </td> </tr> <tr> <th scope="row" style="width: 20%">May 14, 2025</th> <td> <a class="news-title" href="/news/2025-05-14-publication-j-insects-food-feed-bugbook-mating/">New publication: Tomberlin et al. (2025) J Insects Food Feed, in press</a> </td> </tr> <tr> <th scope="row" style="width: 20%">Sep 16, 2024</th> <td> <a class="news-title" href="/news/2024-11-16-publication-carbohydrate-polymers/">New publication: Rodríguez Escobar et al. (2024) Carbohydr Polym 349, 123023</a> </td> </tr> </table> </div> </div> <br> <h3><a href="/blog/" style="color: inherit;">latest posts</a></h3> <div class="news"> <div class="table-responsive"> <table class="table table-sm table-borderless"> <tr> <th scope="row" style="width: 20%">May 17, 2024</th> <td> <a class="news-title" href="/blog/2024/new-COST-action/">New EU-funded COST Action on insect nutrition</a> </td> </tr> <tr> <th scope="row" style="width: 20%">Oct 31, 2023</th> <td> <a class="news-title" href="/blog/2023/end-of-ASEA-UNINET-collaboration/">Our OeAD-funded ASEA-UNINET project Frass4Gas: Thailand comes to an end</a> </td> </tr> <tr> <th scope="row" style="width: 20%">Aug 21, 2023</th> <td> <a class="news-title" href="/blog/2023/end-of-COST-action/">End of EU-funded COST Action CA18131 (ML4Microbiome)</a> </td> </tr> </table> </div> </div> <h3><a href="/publications/" style="color: inherit;">selected publications</a></h3> <div class="publications"> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2020-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2020-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2020-1400.webp"></source> <img src="/assets/img/publication_preview/frontiers_in_microbiology_2020.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="frontiers_in_microbiology_2020.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="klammsteiner_core_2020" class="col-sm-8"> <div class="title">The core gut microbiome of black soldier fly (<i>Hermetia illucens</i>) larvae raised on low-bioburden diets</div> <div class="author"> <em>Thomas Klammsteiner</em>, Andreas Walter, Tajda Bogataj, Carina D. Heussler, Blaž Stres, Florian M. Steiner, Birgit C. Schlick-Steiner, Wolfgang Arthofer, and Heribert Insam</div> <div class="periodical"> <em>Frontiers in Microbiology</em>, May 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="http://doi.org/10.3389/fmicb.2020.00993" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">doi</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/2020-05-21-Klammsteiner-Front-Microbiol.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fmicb.2020.00993"></span> <span class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2020.00993" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>An organism’s gut microbiome handles most of the metabolic processes associated with food intake and digestion but can also strongly affect health and behavior. A stable microbial core community in the gut provides general metabolic competences for substrate degradation and is robust against extrinsic disturbances like changing diets or pathogens. Black Soldier Fly larvae (BSFL; <i>Hermetia illucens</i>) are well known for their ability to efficiently degrade a wide spectrum of organic materials. The ingested substrates build up the high fat and protein content in their bodies that make the larvae interesting for the animal feedstuff industry. In this study, we subjected BSFL to three distinct types of diets carrying a low bioburden and assessed the diets’ impact on larval development and on the composition of the bacterial and archaeal gut community. No significant impact on the gut microbiome across treatments pointed us to the presence of a predominant core community backed by a diverse spectrum of low-abundance taxa. Actinomyces spp., Dysgonomonas spp., and Enterococcus spp. as main members of this community provide various functional and metabolic skills that could be crucial for the thriving of BSFL in various environments. This indicates that the type of diet could play a lesser role in guts of BSFL than previously assumed and that instead a stable autochthonous collection of bacteria provides the tools for degrading of a broad range of substrates. Characterizing the interplay between the core gut microbiome and BSFL helps to understand the involved degradation processes and could contribute to further improving large-scale BSFL rearing.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">klammsteiner_core_2020</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{The core gut microbiome of black soldier fly (&lt;i&gt;Hermetia illucens&lt;/i&gt;) larvae raised on low-bioburden diets}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{11}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License (CC-BY-NC-ND)}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1664-302X}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.frontiersin.org/articles/10.3389/fmicb.2020.00993/full?&amp;utm_source=Email_to_authors_&amp;utm_medium=Email&amp;utm_content=T1_11.5e1_author&amp;utm_campaign=Email_publication&amp;field=&amp;journalName=Frontiers_in_Microbiology&amp;id=499002}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fmicb.2020.00993}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{English}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2020-05-21}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Frontiers in Microbiology}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Klammsteiner, Thomas and Walter, Andreas and Bogataj, Tajda and Heussler, Carina D. and Stres, Blaž and Steiner, Florian M. and Schlick-Steiner, Birgit C. and Arthofer, Wolfgang and Insam, Heribert}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">may</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Frontiers}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{16S amplicon sequencing, Animal feedstuff, Circular economy, microbial communities, Waste valorization, AGIV, Actinomyces, Larval metabolism}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{993}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="nv">true</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2021A-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2021A-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2021A-1400.webp"></source> <img src="/assets/img/publication_preview/frontiers_in_microbiology_2021A.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="frontiers_in_microbiology_2021A.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="klammsteiner_impact_2021" class="col-sm-8"> <div class="title">Impact of processed food (canteen and oil wastes) on the development of black soldier fly (<i>Hermetia illucens</i>) larvae and their gut microbiome functions</div> <div class="author"> <em>Thomas Klammsteiner</em>, Andreas Walter, Tajda Bogataj, Carina D. Heussler, Blaž Stres, Florian M. Steiner, Birgit C. Schlick-Steiner, and Heribert Insam</div> <div class="periodical"> <em>Frontiers in Microbiology</em>, Jan 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="http://doi.org/10.3389/fmicb.2021.619112" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">doi</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/2021-01-21-Klammsteiner-Front-Microbiol.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fmicb.2021.619112"></span> <span class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2021.619112" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Canteens represent an essential food supply hub for educational institutions, companies, and business parks. Many people in these locations rely on a guaranteed service with consistent quality. It is an ongoing challenge to satisfy the demand for sufficient serving numbers, portion sizes, and menu variations to cover food intolerances and different palates of customers. However, overestimating this demand or fluctuating quality of dishes leads to an inevitable loss of unconsumed food due to leftovers. In this study, the food waste fraction of canteen leftovers was identified as an optimal diet for black soldier fly (<i>Hermetia illucens</i>) larvae based on 50% higher consumption and 15% higher waste reduction indices compared with control chicken feed diet. Although the digestibility of food waste was nearly twice as high, the conversion efficiency of ingested and digested chicken feed remains unparalleled (17.9 ± 0.6 and 37.5 ± 0.9 in CFD and 7.9 ± 0.9 and 9.6 ± 1.0 in FWD, respectively). The oil separator waste fraction, however, inhibited biomass gain by at least 85% and ultimately led to a larval mortality of up to 96%. In addition to monitoring larval development, we characterized physicochemical properties of pre- and post-process food waste substrates. High-throughput amplicon sequencing identified Firmicutes, Proteobacteria, and Bacteroidota as the most abundant phyla, and Morganella sp., Acinetobacter sp., and certain Lactobacillales species were identified as indicator species. By using metagenome imputation, we additionally gained insights into the functional spectrum of gut microbial communities. We anticipate that the results will contribute to the development of decentralized waste-management sites that make use of larvae to process food waste as it has become common practice for biogas plants.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">klammsteiner_impact_2021</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Impact of processed food (canteen and oil wastes) on the development of black soldier fly (&lt;i&gt;Hermetia illucens&lt;/i&gt;) larvae and their gut microbiome functions}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{All rights reserved}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1664-302X}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.frontiersin.org/articles/10.3389/fmicb.2021.619112/full}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fmicb.2021.619112}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{English}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2021-02-04}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Frontiers in Microbiology}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Klammsteiner, Thomas and Walter, Andreas and Bogataj, Tajda and Heussler, Carina D. and Stres, Blaž and Steiner, Florian M. and Schlick-Steiner, Birgit C. and Insam, Heribert}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Frontiers}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{16S amplicon sequencing, Animal feedstuff, Circular economy, Growth parameters, Metabolism, microbial communities, Oil waste, Waste valorization}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{619112}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="nv">true</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2021B-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2021B-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2021B-1400.webp"></source> <img src="/assets/img/publication_preview/frontiers_in_microbiology_2021B.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="frontiers_in_microbiology_2021B.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="moreno-indias_statistical_2021" class="col-sm-8"> <div class="title">Statistical and machine learning techniques in human microbiome studies: contemporary challenges and solutions</div> <div class="author"> Isabel Moreno-Indias, Leo Lahti, Miroslava Nedyalkova, Ilze Elbere, Gennady V. Roshchupkin, Muhamed Adilovic, Onder Aydemir, Burcu Bakir-Gungor, Enrique Carrillo-de Santa Pau, Domenica D’Elia, Magesh S. Desai, Laurent Falquet, Aycan Gundogdu, Karel Hron, <em>Thomas Klammsteiner</em>, Marta B. Lopes, Laura Judith Marcos Zambrano, Cláudia Marques, Michael Mason, Patrick May, Lejla Pašić, Gianvito Pio, Sándor Pongor, Vasilis J. Promponas, Piotr Przymus, Julio Sáez-Rodríguez, Alexia Sampri, Rajesh Shigdel, Blaz Stres, Ramona Suharoschi, Jaak Truu, Ciprian-Octavian Truică, Baiba Vilne, Dimitrios P. Vlachakis, Ercüment Yılmaz, Georg Zeller, Aldert Zomer, David Gómez-Cabrero, and Marcus Claesson</div> <div class="periodical"> <em>Frontiers in Microbiology</em>, Feb 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="http://doi.org/10.3389/fmicb.2021.635781" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">doi</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/2021-02-22-Moreno-Indias-Front-Microbiol.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fmicb.2021.635781"></span> <span class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2021.635781" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Human microbiome has emerged as a central research topic in human biology and biomedicine. Current microbiome studies generate high-throughput omics data across different body sites, populations, and lifetime. Whereas many of the challenges in microbiome research are similar to other high-throughput studies, the quantitative analyses need to address the heterogeneity of data, specific statistical properties, and the remarkable variation in microbiome composition across individuals and body sites. This has led to a broad spectrum of statistical and machine learning challenges that range from study design, data processing, and standardization to analysis, modeling, cross-study comparison, prediction, data science ecosystems, and reproducible reporting. Nevertheless, although many statistics and machine learning approaches and tools have been developed, new techniques are needed to deal with emerging applications and the vast heterogeneity of microbiome data. We review and discuss emerging applications of statistical and machine learning techniques in human microbiome studies and introduce the COST Action CA18131 "ML4Microbiome" that brings together microbiome researchers and machine learning experts to address current challenges such as standardisation of analysis pipelines for reproducibility of data analysis results, benchmarking, improvement, or development of existing and new tools and ontologies.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">moreno-indias_statistical_2021</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Statistical and machine learning techniques in human microbiome studies: contemporary challenges and solutions}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{All rights reserved}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1664-302X}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{Statistical and machine learning techniques in human microbiome studies}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.frontiersin.org/articles/10.3389/fmicb.2021.635781/abstract}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fmicb.2021.635781}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{English}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2021-02-18}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Frontiers in Microbiology}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Moreno-Indias, Isabel and Lahti, Leo and Nedyalkova, Miroslava and Elbere, Ilze and Roshchupkin, Gennady V. and Adilovic, Muhamed and Aydemir, Onder and Bakir-Gungor, Burcu and Carrillo-de Santa Pau, Enrique and D'Elia, Domenica and Desai, Magesh S. and Falquet, Laurent and Gundogdu, Aycan and Hron, Karel and Klammsteiner, Thomas and Lopes, Marta B. and Marcos Zambrano, Laura Judith and Marques, Cláudia and Mason, Michael and May, Patrick and Pašić, Lejla and Pio, Gianvito and Pongor, Sándor and Promponas, Vasilis J. and Przymus, Piotr and Sáez-Rodríguez, Julio and Sampri, Alexia and Shigdel, Rajesh and Stres, Blaz and Suharoschi, Ramona and Truu, Jaak and Truică, Ciprian-Octavian and Vilne, Baiba and Vlachakis, Dimitrios P. and Yılmaz, Ercüment and Zeller, Georg and Zomer, Aldert and Gómez-Cabrero, David and Claesson, Marcus}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">feb</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Frontiers}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{microbiome, biomarker identification, machine learning, ML4microbiome, personalized medicine}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{635781}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="nv">true</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2023C-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2023C-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2023C-1400.webp"></source> <img src="/assets/img/publication_preview/frontiers_in_microbiology_2023C.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="frontiers_in_microbiology_2023C.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="marcos_zambrano_toolbox_2023" class="col-sm-8"> <div class="title">A toolbox of machine learning software to support microbiome analysis</div> <div class="author"> Laura Judith Marcos Zambrano, Víctor Manuel López Molina, Burcu Bakir-Gungor*, Marcus Frohme*, Kanita Karaduzovic-Hadziabdic*, <em>Thomas Klammsteiner*</em>, Eliana Ibrahimi*, Leo Lahti*, Tatjana Loncar-Turukalo*, Xhilda Dhamo*, Andrea Simeon*, Alina Nechyporenko*, Gianvito Pio*, Piotr Przymus*, Alexia Sampri*, Vladimir Tihomir Trajkovik*, Oliver Aasmets, Ricardo Araujo, Ioannis Anagnostopoulos, Onder Aydemir, Magali Berland, María de la Luz Calle, Michelangelo Ceci, Hatice Duman, Aycan Gundogdu, Aki S. Havulinna, Kardokh Hama Najib Kaka Bra, Eglantina Kalluci, Sercan Karav, Daniel Lode, Marta B. Lopes, Patrick May, Bram Nap, Miroslava Nedyalkova, Inês Paciência, Lejla Pasic, Meritxell Pujolassos, Rajesh Shigdel, Antonio Susin, Ines Thiele, Ciprian-Octavian Truică, Paul Wilmes, Ercüment Yılmaz, Malik Yousef, Marcus Joakim Claesson, Jaak Truu, and Enrique Carrillo De Santa Pau</div> <div class="periodical"> <em>Frontiers in Microbiology</em>, Nov 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="http://doi.org/10.3389/fmicb.2023.1250806" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">doi</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/2023-11-22-Marcos-Zambrano-Front-Microbiol.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2023.1250806" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The human microbiome has become an area of intense research due to its potential impact on human health. However, the analysis and interpretation of this data have proven to be challenging due to its complexity and high dimensionality. Machine learning (ML) algorithms can process vast amounts of data to uncover informative patterns and relationships within the data, even with limited prior knowledge. Therefore, there has been a rapid growth in the development of software specifically designed for the analysis and interpretation of microbiome data using ML techniques. These software incorporate a wide range of ML algorithms for clustering, classification, regression, or feature selection, to identify microbial patterns and relationships within the data and generate predictive models. This rapid development with a constant need for new developments and integration of new features require efforts into compile, catalog and classify these tools to create infrastructures and services with easy, transparent, and trustable standards. Here we review the state-of-the-art for ML tools applied in human microbiome studies, performed as part of the COST Action ML4Microbiome activities. This scoping review focuses on ML based software and framework resources currently available for the analysis of microbiome data in humans. The aim is to support microbiologists and biomedical scientists to go deeper into specialized resources that integrate ML techniques and facilitate future benchmarking to create standards for the analysis of microbiome data. The software resources are organized based on the type of analysis they were developed for and the ML techniques they implement. A description of each software with examples of usage is provided including comments about pitfalls and lacks in the usage of software based on ML methods in relation to microbiome data that need to be considered by developers and users. This review represents an extensive compilation to date, offering valuable insights and guidance for researchers interested in leveraging ML approaches for microbiome analysis.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">marcos_zambrano_toolbox_2023</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A toolbox of machine learning software to support microbiome analysis}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{14}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{All rights reserved}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1664-302X}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.frontiersin.org/articles/10.3389/fmicb.2023.1250806}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fmicb.2023.1250806}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{English}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2023-11-12}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Frontiers in Microbiology}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Marcos Zambrano, Laura Judith and López Molina, Víctor Manuel and Bakir-Gungor*, Burcu and Frohme*, Marcus and Karaduzovic-Hadziabdic*, Kanita and Klammsteiner*, Thomas and Ibrahimi*, Eliana and Lahti*, Leo and Loncar-Turukalo*, Tatjana and Dhamo*, Xhilda and Simeon*, Andrea and Nechyporenko*, Alina and Pio*, Gianvito and Przymus*, Piotr and Sampri*, Alexia and Trajkovik*, Vladimir Tihomir and Aasmets, Oliver and Araujo, Ricardo and Anagnostopoulos, Ioannis and Aydemir, Onder and Berland, Magali and Calle, María de la Luz and Ceci, Michelangelo and Duman, Hatice and Gundogdu, Aycan and Havulinna, Aki S. and Kaka Bra, Kardokh Hama Najib and Kalluci, Eglantina and Karav, Sercan and Lode, Daniel and Lopes, Marta B. and May, Patrick and Nap, Bram and Nedyalkova, Miroslava and Paciência, Inês and Pasic, Lejla and Pujolassos, Meritxell and Shigdel, Rajesh and Susin, Antonio and Thiele, Ines and Truică, Ciprian-Octavian and Wilmes, Paul and Yılmaz, Ercüment and Yousef, Malik and Claesson, Marcus Joakim and Truu, Jaak and Carrillo De Santa Pau, Enrique}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">nov</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Frontiers}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1250806}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="nv">true</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/iscience_2025-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/iscience_2025-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/iscience_2025-1400.webp"></source> <img src="/assets/img/publication_preview/iscience_2025.png" class="preview z-depth-1 rounded" width="auto" height="auto" alt="iscience_2025.png" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="klammsteiner_iscience_2025" class="col-sm-8"> <div class="title">Larval density drives thermogenesis and affects microbiota and substrate properties in black soldier fly trials</div> <div class="author"> <em>Thomas Klammsteiner</em>, Carina D. Heussler, Heribert Insam, Birgit C. Schlick-Steiner, and Florian M. Steiner</div> <div class="periodical"> <em>iScience</em>, May 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="http://doi.org/10.1016/j.isci.2025.112794" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">doi</a> <a href="/assets/pdf/2025-05-30-Klammsteiner-iScience.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.isci.2025.112794" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Industrial insect farming has potential for converting low-value organic waste into nutrient-rich insect biomass, producing valuable by-products like organic fertilizers. However, a better understanding of density-related thermogenesis and microbial dynamics is needed to enhance standardization and bridge gaps between laboratory and industry needs. This lab-scale study focuses on the black soldier fly (Hermetia illucens), which exhibits thermogenesis that intensifies with larval population size and its natural crowding behavior. Using high-resolution temperature monitoring and biomolecular methods, we found that doubling larval density (0, 1.25, 2.5, 5 larvae/cm2) increased temperatures by 0.6 to 2.4 °C, depending on the treatment. Adding potassium sorbate altered microbial profiles, increasing Enterobacter and decreasing Providencia, while promoting lactic acid bacteria. Density also impacted pH, water content, dry matter, volatile solids, and ash in the substrate. Our findings provide essential insights into managing microbial and thermal dynamics, offering valuable information for optimizing and standardizing conditions in rearing trials.</p> </div> </div> </div> </li> </ol> </div> </article> </div> </div> <footer class="sticky-bottom mt-5"> <div class="container"> © Copyright 2025 Thomas Klammsteiner. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?7b30caa5023af4af8408a472dc4e1ebb"></script> <script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js?d633890033921b33e0ceb13d22340a9c"></script> <script defer src="/assets/js/common.js?acdb9690d7641b2f8d40529018c71a01"></script> <script defer src="/assets/js/copy_code.js?c9d9dd48933de3831b3ee5ec9c209cac" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>