<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>ML4Microbiome | Thomas Klammsteiner</title> <meta name="author" content="Thomas Klammsteiner"> <meta name="description" content="COST Action CA18131 Statistical and machine learning techniques in human microbiome studies"> <meta name="keywords" content="microbiome, data science, microbial communities, insect biotechnology, insects, waste management, anaerobic digestion, bioinformatics"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?19f3075a2d19613090fe9e16b564e1fe" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/icon.jpg?4ad390f64e9fd1b08f8da0b3b05a4fcb"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://tklammsteiner.github.io/projects/ML4MICROBIOME/"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?e74e74bf055e5729d44a7d031a5ca6a5" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?96d6b3e1c3604aca8b6134c7afdd5db6"></script> <script src="/assets/js/dark_mode.js?9b17307bb950ffa2e34be0227f53558f"></script> </head> <body class="fixed-top-nav sticky-bottom-footer"> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Thomas </span>Klammsteiner</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog</a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications</a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects</a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">ML4Microbiome</h1> <p class="post-description">COST Action CA18131 Statistical and machine learning techniques in human microbiome studies</p> </header> <article> <h3 id="project-details"><strong>Project details</strong></h3> <hr> <p><strong>Funding agency:</strong> COST European Cooperation in Science &amp; Technology<br> <strong>Period:</strong> Feb. 2019 - Aug. 2023<br> <strong>Function:</strong> Management Committee Member<br> <strong>Link:</strong> https://www.cost.eu/actions/CA18131/</p> <p>ML4Microbiome website: <a href="https://www.ml4microbiome.eu/" rel="external nofollow noopener" target="_blank">https://www.ml4microbiome.eu/</a></p> </article> <h2>References</h2> <div class="publications"> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2023B-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2023B-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2023B-1400.webp"></source> <img src="/assets/img/publication_preview/frontiers_in_microbiology_2023B.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="frontiers_in_microbiology_2023B.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="papoutsoglou_machine_2023" class="col-sm-8"> <div class="title">Machine learning approaches in microbiome research: challenges and best practices</div> <div class="author"> Georgios Papoutsoglou, Sonia Tarazona, Marta B. Lopes, <em>Thomas Klammsteiner</em>, Eliana Ibrahimi, Julia Eckenberger, Pierfrancesco Novielli, Alberto Tonda, Andrea Simeon, Rajesh Shigdel, Stéphane Béreux, Giacomo Vitali, Sabina Tangaro, Leo Lahti, Andriy Temko, Marcus J. Claesson, and Magali Berland</div> <div class="periodical"> <em>Frontiers in Microbiology</em>, Dec 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="http://doi.org/10.3389/fmicb.2023.1261889" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">doi</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/2023-09-22-Papoutsoglou-Front-Microbiol.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fmicb.2023.1261889"></span> <span class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2023.1261889" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Microbiome data predictive analysis within a machine learning (ML) workflow presents numerous domain-specific challenges involving preprocessing, feature selection, predictive modeling, performance estimation, model interpretation, and the extraction of biological information from the results. To assist decision-making, we offer a set of recommendations on algorithm selection, pipeline creation and evaluation, stemming from the COST Action ML4Microbiome. We compared the suggested approaches on a multi-cohort shotgun metagenomics dataset of colorectal cancer patients, focusing on their performance in disease diagnosis and biomarker discovery. It is demonstrated that the use of compositional transformations and filtering methods as part of data preprocessing does not always improve the predictive performance of a model. In contrast, the multivariate feature selection, such as the Statistically Equivalent Signatures algorithm, was effective in reducing the classification error. When validated on a separate test dataset, this algorithm in combination with random forest modeling, provided the most accurate performance estimates. Lastly, we showed how linear modeling by logistic regression coupled with visualization techniques such as Individual Conditional Expectation (ICE) plots can yield interpretable results and offer biological insights. These findings are significant for clinicians and non-experts alike in translational applications.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">papoutsoglou_machine_2023</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Machine learning approaches in microbiome research: challenges and best practices}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{14}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{All rights reserved}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1664-302X}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{Machine learning approaches in microbiome research}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.frontiersin.org/articles/10.3389/fmicb.2023.1261889}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fmicb.2023.1261889}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2023-09-22}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Frontiers in Microbiology}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Papoutsoglou, Georgios and Tarazona, Sonia and Lopes, Marta B. and Klammsteiner, Thomas and Ibrahimi, Eliana and Eckenberger, Julia and Novielli, Pierfrancesco and Tonda, Alberto and Simeon, Andrea and Shigdel, Rajesh and Béreux, Stéphane and Vitali, Giacomo and Tangaro, Sabina and Lahti, Leo and Temko, Andriy and Claesson, Marcus J. and Berland, Magali}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Frontiers}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="nv">true</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2023A-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2023A-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2023A-1400.webp"></source> <img src="/assets/img/publication_preview/frontiers_in_microbiology_2023A.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="frontiers_in_microbiology_2023A.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="delia_advancing_2023" class="col-sm-8"> <div class="title">Advancing microbiome research with machine learning: key findings from the ML4Microbiome COST action</div> <div class="author"> Domenica D’Elia, Jaak Truu, Leo Lahti, Magali Berland, Georgios Papoutsoglou, Michelangelo Ceci, Aldert Zomer, Marta B. Lopes, Eliana Ibrahimi, Aleksandra Gruca, Alina Nechyporenko, Marcus Frohme, <em>Thomas Klammsteiner</em>, Enrique Carrillo-de Santa Pau, Laura Judith Marcos-Zambrano, Karel Hron, Gianvito Pio, Andrea Simeon, Ramona Suharoschi, Isabel Moreno-Indias, Andriy Temko, Miroslava Nedyalkova, Elena-Simona Apostol, Ciprian-Octavian Truică, Rajesh Shigdel, Jasminka Hasić Telalović, Erik Bongcam-Rudloff, Piotr Przymus, Naida Babić Jordamović, Laurent Falquet, Sonia Tarazona, Alexia Sampri, Gaetano Isola, David Pérez-Serrano, Vladimir Trajkovik, Lubos Klucar, Tatjana Loncar-Turukalo, Aki S. Havulinna, Christian Jansen, Randi J. Bertelsen, and Marcus Joakim Claesson</div> <div class="periodical"> <em>Frontiers in Microbiology</em>, Dec 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="http://doi.org/10.3389/fmicb.2023.1257002" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">doi</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/2023-09-25-DElia-Front-Microbiol.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fmicb.2023.1257002"></span> <span class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2023.1257002" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The rapid development of machine learning (ML) techniques has opened up the data-dense field of microbiome research for novel therapeutic, diagnostic, and prognostic applications targeting a wide range of disorders, which could substantially improve healthcare practices in the era of precision medicine. However, several challenges must be addressed to exploit the benefits of ML in this field fully. In particular, there is a need to establish “gold standard” protocols for conducting ML analysis experiments and improve interactions between microbiome researchers and ML experts. The Machine Learning Techniques in Human Microbiome Studies (ML4Microbiome) COST Action CA18131 is a European network established in 2019 to promote collaboration between discovery-oriented microbiome researchers and data-driven ML experts to optimize and standardize ML approaches for microbiome analysis. This perspective paper presents the key achievements of ML4Microbiome, which include identifying predictive and discriminatory ‘omics’ features, improving repeatability and comparability, developing automation procedures, and defining priority areas for the novel development of ML methods targeting the microbiome. The insights gained from ML4Microbiome will help to maximize the potential of ML in microbiome research and pave the way for new and improved healthcare practices.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">delia_advancing_2023</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Advancing microbiome research with machine learning: key findings from the ML4Microbiome COST action}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{14}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{All rights reserved}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1664-302X}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{Advancing microbiome research with machine learning}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.frontiersin.org/articles/10.3389/fmicb.2023.1257002}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fmicb.2023.1257002}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2023-09-21}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Frontiers in Microbiology}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{D’Elia, Domenica and Truu, Jaak and Lahti, Leo and Berland, Magali and Papoutsoglou, Georgios and Ceci, Michelangelo and Zomer, Aldert and Lopes, Marta B. and Ibrahimi, Eliana and Gruca, Aleksandra and Nechyporenko, Alina and Frohme, Marcus and Klammsteiner, Thomas and Pau, Enrique Carrillo-de Santa and Marcos-Zambrano, Laura Judith and Hron, Karel and Pio, Gianvito and Simeon, Andrea and Suharoschi, Ramona and Moreno-Indias, Isabel and Temko, Andriy and Nedyalkova, Miroslava and Apostol, Elena-Simona and Truică, Ciprian-Octavian and Shigdel, Rajesh and Telalović, Jasminka Hasić and Bongcam-Rudloff, Erik and Przymus, Piotr and Jordamović, Naida Babić and Falquet, Laurent and Tarazona, Sonia and Sampri, Alexia and Isola, Gaetano and Pérez-Serrano, David and Trajkovik, Vladimir and Klucar, Lubos and Loncar-Turukalo, Tatjana and Havulinna, Aki S. and Jansen, Christian and Bertelsen, Randi J. and Claesson, Marcus Joakim}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Frontiers}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="nv">true</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2023C-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2023C-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2023C-1400.webp"></source> <img src="/assets/img/publication_preview/frontiers_in_microbiology_2023C.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="frontiers_in_microbiology_2023C.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="marcos_zambrano_toolbox_2023" class="col-sm-8"> <div class="title">A toolbox of machine learning software to support microbiome analysis</div> <div class="author"> Laura Judith Marcos Zambrano, Víctor Manuel López Molina, Burcu Bakir-Gungor*, Marcus Frohme*, Kanita Karaduzovic-Hadziabdic*, <em>Thomas Klammsteiner*</em>, Eliana Ibrahimi*, Leo Lahti*, Tatjana Loncar-Turukalo*, Xhilda Dhamo*, Andrea Simeon*, Alina Nechyporenko*, Gianvito Pio*, Piotr Przymus*, Alexia Sampri*, Vladimir Tihomir Trajkovik*, Oliver Aasmets, Ricardo Araujo, Ioannis Anagnostopoulos, Onder Aydemir, Magali Berland, María de la Luz Calle, Michelangelo Ceci, Hatice Duman, Aycan Gundogdu, Aki S. Havulinna, Kardokh Hama Najib Kaka Bra, Eglantina Kalluci, Sercan Karav, Daniel Lode, Marta B. Lopes, Patrick May, Bram Nap, Miroslava Nedyalkova, Inês Paciência, Lejla Pasic, Meritxell Pujolassos, Rajesh Shigdel, Antonio Susin, Ines Thiele, Ciprian-Octavian Truică, Paul Wilmes, Ercüment Yılmaz, Malik Yousef, Marcus Joakim Claesson, Jaak Truu, and Enrique Carrillo De Santa Pau</div> <div class="periodical"> <em>Frontiers in Microbiology</em>, Nov 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="http://doi.org/10.3389/fmicb.2023.1250806" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">doi</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/2023-11-22-Marcos-Zambrano-Front-Microbiol.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2023.1250806" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The human microbiome has become an area of intense research due to its potential impact on human health. However, the analysis and interpretation of this data have proven to be challenging due to its complexity and high dimensionality. Machine learning (ML) algorithms can process vast amounts of data to uncover informative patterns and relationships within the data, even with limited prior knowledge. Therefore, there has been a rapid growth in the development of software specifically designed for the analysis and interpretation of microbiome data using ML techniques. These software incorporate a wide range of ML algorithms for clustering, classification, regression, or feature selection, to identify microbial patterns and relationships within the data and generate predictive models. This rapid development with a constant need for new developments and integration of new features require efforts into compile, catalog and classify these tools to create infrastructures and services with easy, transparent, and trustable standards. Here we review the state-of-the-art for ML tools applied in human microbiome studies, performed as part of the COST Action ML4Microbiome activities. This scoping review focuses on ML based software and framework resources currently available for the analysis of microbiome data in humans. The aim is to support microbiologists and biomedical scientists to go deeper into specialized resources that integrate ML techniques and facilitate future benchmarking to create standards for the analysis of microbiome data. The software resources are organized based on the type of analysis they were developed for and the ML techniques they implement. A description of each software with examples of usage is provided including comments about pitfalls and lacks in the usage of software based on ML methods in relation to microbiome data that need to be considered by developers and users. This review represents an extensive compilation to date, offering valuable insights and guidance for researchers interested in leveraging ML approaches for microbiome analysis.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">marcos_zambrano_toolbox_2023</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A toolbox of machine learning software to support microbiome analysis}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{14}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{All rights reserved}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1664-302X}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.frontiersin.org/articles/10.3389/fmicb.2023.1250806}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fmicb.2023.1250806}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{English}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2023-11-12}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Frontiers in Microbiology}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Marcos Zambrano, Laura Judith and López Molina, Víctor Manuel and Bakir-Gungor*, Burcu and Frohme*, Marcus and Karaduzovic-Hadziabdic*, Kanita and Klammsteiner*, Thomas and Ibrahimi*, Eliana and Lahti*, Leo and Loncar-Turukalo*, Tatjana and Dhamo*, Xhilda and Simeon*, Andrea and Nechyporenko*, Alina and Pio*, Gianvito and Przymus*, Piotr and Sampri*, Alexia and Trajkovik*, Vladimir Tihomir and Aasmets, Oliver and Araujo, Ricardo and Anagnostopoulos, Ioannis and Aydemir, Onder and Berland, Magali and Calle, María de la Luz and Ceci, Michelangelo and Duman, Hatice and Gundogdu, Aycan and Havulinna, Aki S. and Kaka Bra, Kardokh Hama Najib and Kalluci, Eglantina and Karav, Sercan and Lode, Daniel and Lopes, Marta B. and May, Patrick and Nap, Bram and Nedyalkova, Miroslava and Paciência, Inês and Pasic, Lejla and Pujolassos, Meritxell and Shigdel, Rajesh and Susin, Antonio and Thiele, Ines and Truică, Ciprian-Octavian and Wilmes, Paul and Yılmaz, Ercüment and Yousef, Malik and Claesson, Marcus Joakim and Truu, Jaak and Carrillo De Santa Pau, Enrique}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">nov</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Frontiers}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1250806}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="nv">true</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2021C-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2021C-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2021C-1400.webp"></source> <img src="/assets/img/publication_preview/frontiers_in_microbiology_2021C.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="frontiers_in_microbiology_2021C.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="zambrano_applications_2021" class="col-sm-8"> <div class="title">Applications of machine learning in human microbiome studies: a review on feature selection, biomarker identification, disease prediction and treatment</div> <div class="author"> Laura Judith Marcos Zambrano, Kanita Karaduzovic-Hadziabdic, Tatjana Loncar-Turukalo, Piotr Przymus, Vladimir Trajkovik, Oliver Aasmets, Magali Berland, Aleksandra Gruca, Jasminka Hasic Telalovic, Hron Karel, <em>Thomas Klammsteiner</em>, Mikhail Kolev, Leo Lahti, Mart B. Lopes, Victor Moreno, Irina Naskinova, Elin Org, Inês Paciência, Georgios Papoutsoglou, Rajesh Shigdel, Blaz Stres, Baiba Vilne, Malik Yousef, Eftim Zdravevski, Ioannis Tsamardinos, Enrique Carrillo Santa Pau, Marcus Claesson, Isabel Moreno Indias, and Jaak Truu</div> <div class="periodical"> <em>Frontiers in Microbiology</em>, Feb 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="http://doi.org/10.3389/fmicb.2021.634511" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">doi</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/2021-02-19-Marcos-Zambrano-Front-Microbiol.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fmicb.2021.634511"></span> <span class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2021.634511" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The number of microbiome-related studies has notably increased the availability of data on human microbiome composition and function. These studies provide the essential material to deeply explore host-microbiome associations and their relation to the development and progression of various complex diseases. Improved data-analytical tools are needed to exploit all information from these biological datasets, taking into account the peculiarities of microbiome data, i.e. compositional, heterogeneous and sparse nature of these datasets. The possibility of predicting host-phenotypes based on taxonomy-informed feature selection to establish an association between microbiome and predict disease states is beneficial for personalized medicine. In this regard, machine learning (ML) provides new insights into the development of models that can be used to predict outputs, such as classification and prediction in microbiology, infer host phenotypes to predict diseases and use microbial communities to stratify patients by their characterization of state-specific microbial signatures. Here we review the state-of-the-art ML methods and respective software applied in human microbiome studies, performed as part of the COST Action ML4Microbiome activities. This scoping review focuses on the application of ML in microbiome studies related to association and clinical use for diagnostics, prognostics, and therapeutics. Although the data presented here is more related to the bacterial community, many algorithms could be applied in general, regardless of the feature type. This literature and software review covering this broad topic is aligned with the scoping review methodology. The manual identification of data sources has been complemented with: (1) automated publication search through digital libraries of the three major publishers using natural language processing (NLP) Toolkit, and (2) an automated identification of relevant software repositories on GitHub and ranking of the related research papers relying on learning to rank approach.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">zambrano_applications_2021</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Applications of machine learning in human microbiome studies: a review on feature selection, biomarker identification, disease prediction and treatment}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{All rights reserved}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1664-302X}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{Applications of machine learning in human microbiome studies}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.frontiersin.org/articles/10.3389/fmicb.2021.634511/abstract}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fmicb.2021.634511}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{English}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2021-02-17}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Frontiers in Microbiology}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zambrano, Laura Judith Marcos and Karaduzovic-Hadziabdic, Kanita and Loncar-Turukalo, Tatjana and Przymus, Piotr and Trajkovik, Vladimir and Aasmets, Oliver and Berland, Magali and Gruca, Aleksandra and Hasic Telalovic, Jasminka and Karel, Hron and Klammsteiner, Thomas and Kolev, Mikhail and Lahti, Leo and Lopes, Mart B. and Moreno, Victor and Naskinova, Irina and Org, Elin and Paciência, Inês and Papoutsoglou, Georgios and Shigdel, Rajesh and Stres, Blaz and Vilne, Baiba and Yousef, Malik and Zdravevski, Eftim and Tsamardinos, Ioannis and de Santa Pau, Enrique Carrillo and Claesson, Marcus and Moreno Indias, Isabel and Truu, Jaak}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">feb</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Frontiers}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{microbiome, biomarker identification, Disease Prediction, feature  selection, machine   learning}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{634511}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2021B-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2021B-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/frontiers_in_microbiology_2021B-1400.webp"></source> <img src="/assets/img/publication_preview/frontiers_in_microbiology_2021B.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="frontiers_in_microbiology_2021B.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="moreno-indias_statistical_2021" class="col-sm-8"> <div class="title">Statistical and machine learning techniques in human microbiome studies: contemporary challenges and solutions</div> <div class="author"> Isabel Moreno-Indias, Leo Lahti, Miroslava Nedyalkova, Ilze Elbere, Gennady V. Roshchupkin, Muhamed Adilovic, Onder Aydemir, Burcu Bakir-Gungor, Enrique Carrillo-de Santa Pau, Domenica D’Elia, Magesh S. Desai, Laurent Falquet, Aycan Gundogdu, Karel Hron, <em>Thomas Klammsteiner</em>, Marta B. Lopes, Laura Judith Marcos Zambrano, Cláudia Marques, Michael Mason, Patrick May, Lejla Pašić, Gianvito Pio, Sándor Pongor, Vasilis J. Promponas, Piotr Przymus, Julio Sáez-Rodríguez, Alexia Sampri, Rajesh Shigdel, Blaz Stres, Ramona Suharoschi, Jaak Truu, Ciprian-Octavian Truică, Baiba Vilne, Dimitrios P. Vlachakis, Ercüment Yılmaz, Georg Zeller, Aldert Zomer, David Gómez-Cabrero, and Marcus Claesson</div> <div class="periodical"> <em>Frontiers in Microbiology</em>, Feb 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="http://doi.org/10.3389/fmicb.2021.635781" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">doi</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/2021-02-22-Moreno-Indias-Front-Microbiol.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fmicb.2021.635781"></span> <span class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2021.635781" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Human microbiome has emerged as a central research topic in human biology and biomedicine. Current microbiome studies generate high-throughput omics data across different body sites, populations, and lifetime. Whereas many of the challenges in microbiome research are similar to other high-throughput studies, the quantitative analyses need to address the heterogeneity of data, specific statistical properties, and the remarkable variation in microbiome composition across individuals and body sites. This has led to a broad spectrum of statistical and machine learning challenges that range from study design, data processing, and standardization to analysis, modeling, cross-study comparison, prediction, data science ecosystems, and reproducible reporting. Nevertheless, although many statistics and machine learning approaches and tools have been developed, new techniques are needed to deal with emerging applications and the vast heterogeneity of microbiome data. We review and discuss emerging applications of statistical and machine learning techniques in human microbiome studies and introduce the COST Action CA18131 "ML4Microbiome" that brings together microbiome researchers and machine learning experts to address current challenges such as standardisation of analysis pipelines for reproducibility of data analysis results, benchmarking, improvement, or development of existing and new tools and ontologies.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">moreno-indias_statistical_2021</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Statistical and machine learning techniques in human microbiome studies: contemporary challenges and solutions}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{All rights reserved}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1664-302X}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{Statistical and machine learning techniques in human microbiome studies}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.frontiersin.org/articles/10.3389/fmicb.2021.635781/abstract}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fmicb.2021.635781}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{English}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2021-02-18}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Frontiers in Microbiology}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Moreno-Indias, Isabel and Lahti, Leo and Nedyalkova, Miroslava and Elbere, Ilze and Roshchupkin, Gennady V. and Adilovic, Muhamed and Aydemir, Onder and Bakir-Gungor, Burcu and Carrillo-de Santa Pau, Enrique and D'Elia, Domenica and Desai, Magesh S. and Falquet, Laurent and Gundogdu, Aycan and Hron, Karel and Klammsteiner, Thomas and Lopes, Marta B. and Marcos Zambrano, Laura Judith and Marques, Cláudia and Mason, Michael and May, Patrick and Pašić, Lejla and Pio, Gianvito and Pongor, Sándor and Promponas, Vasilis J. and Przymus, Piotr and Sáez-Rodríguez, Julio and Sampri, Alexia and Shigdel, Rajesh and Stres, Blaz and Suharoschi, Ramona and Truu, Jaak and Truică, Ciprian-Octavian and Vilne, Baiba and Vlachakis, Dimitrios P. and Yılmaz, Ercüment and Zeller, Georg and Zomer, Aldert and Gómez-Cabrero, David and Claesson, Marcus}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">feb</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Frontiers}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{microbiome, biomarker identification, machine learning, ML4microbiome, personalized medicine}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{635781}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="nv">true</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> </div> </div> </div> <footer class="sticky-bottom mt-5"> <div class="container"> © Copyright 2025 Thomas Klammsteiner. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?7b30caa5023af4af8408a472dc4e1ebb"></script> <script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js?d633890033921b33e0ceb13d22340a9c"></script> <script defer src="/assets/js/common.js?acdb9690d7641b2f8d40529018c71a01"></script> <script defer src="/assets/js/copy_code.js?c9d9dd48933de3831b3ee5ec9c209cac" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>